### PR TITLE
Changes for azure UI and new azure assets icons

### PR DESCRIPTION
**Pull request by SauarvD**
### Requirements

* Validated and added respective changes for azure in UI
* Added new azure assets icons for the same

### CODE REVIEW

File: core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.html [hunk 1]
```diff
@@ -47,21 +47,57 @@
...
+                                    <img class='server-icon' src='./../../../assets/images/Azure_Database.svg'>
+                                </span>
+                                <span *ngSwitchCase='"storage_blob_container"'>
+                                    <img class='server-icon' src='./../../../assets/images/Blob_Storage.svg'>
+                                </span>
+                                <span *ngSwitchCase='"cosmosdb_collection"'>
+                                    <img class='server-icon' src='./../../../assets/images/Virtual_Clusters.svg'>
+                                </span>
+                                <span *ngSwitchCase='"eventhubs_namespace"'>
+                                    <img class='server-icon' src='./../../../assets/images/Event_Grid_Domains.svg'>
```
Comment (devsatishm) [inline core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.html]: Let's follow a convention on how we name these files and their locations. Suggestion -> `../assets/images/azure/cosmos_db.svg`, `../assets/images/aws/lambda.svg`, `../assets/images/azure/service_bus_relay.svg` etc.

File: core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts [hunk 2]
```diff
@@ -322,6 +322,7 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
             'sqs',
             'kinesis_stream',
             'apigee_proxy',
+            'storage_account'
```
[the remark this answers is not in the document]
Comment (SauarvD) [reply core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts]: as per the docs I have shared in the channel and the metrics.json that was previously written by other team, only storage-account is being supported for azure metrics

[the remark this answers is not in the document]
Comment (raghits) [reply core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts]: Metrics for any of the Azure services is provided through the associated storage account. And the UI sends the storage account to get the metrics from the metrics API.

File: core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.html (hunk 1 above)
Comment (raghits) [reply core/jazz_ui/src/app/pages/environment-assets/env-assets-section.component.html]: Let's have a separate story to re-organize these assets.

**Pull request merged**